crates/router/src/routes/webhooks.rs

```diff
@@ -44,6 +45,86 @@ pub async fn receive_incoming_webhook<W: types::OutgoingWebhookType>(
     .await
 }
 
+#[cfg(feature = "v1")]
+#[instrument(skip_all, fields(flow = ?Flow::IncomingRelayWebhookReceive))]
+pub async fn receive_incoming_relay_webhook<W: types::OutgoingWebhookType>(
+    state: web::Data<AppState>,
+    req: HttpRequest,
+    body: web::Bytes,
+    path: web::Path<(common_utils::id_type::MerchantId, String)>,
```

Comment:
Can we use `MerchantConnectorAccountId` instead of a `String`, if we only expect a MCA ID in the path?

```suggestion
    path: web::Path<(common_utils::id_type::MerchantId, common_utils::id_type::MerchantConnectorAccountId)>,
```